test/core/fixtures/transformation/es7.function-bind/call/expected.js

```diff
@@ -0,0 +1,8 @@
+"use strict";
+
+var _context;
+
+ns.obj.func.call(ctx);
+(_context = ns.obj).func.call(_context);
+
+ns.obj1.func.call(ns.obj2);
```

Comment:
@zenparsing 
> It might need to translate to something like:

> ```javascript
(_context = ns.obj2, ns).obj1.func.call(_context.prop);
```

Shouldn't it be

```javascript
(_context = ns.obj2.prop, ns).obj1.func.call(_context);
```

instead so that `obj2.prop` is also accessed earlier than `obj1`?